packages/react-components/react-checkbox/library/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -27,7 +27,7 @@ export type CheckboxSlots = {
   /**
    * The checkbox, with the checkmark icon as its child when checked.
    */
-  indicator: Slot<'div'>;
+  indicator: Slot<'span'>;
```

Comment:
```suggestion
  indicator: Slot<'div', 'span'>;
```

This will be a breaking change, unfortunately we cannot do that in a minor version 🥲

Reply:
Thanks @bsunderhus ❤️ I expected beachball to have a "major" option, but it seems it's turned off on purpose. How we will make sure that it won't be forgotten during next major version release?